src/main/java/frc/robot/RobotContainer.java

```diff
@@ -33,19 +30,13 @@
 import frc.robot.commands.drivetrain.SetInitOdom;
 import frc.robot.commands.feeder.StartFeeder;
 import frc.robot.commands.intake.StartIntake;
+import frc.robot.commands.pathplanning.PathPlannerShoot;
 import frc.robot.commands.ramp.RampMove;
 import frc.robot.commands.ramp.ResetRamp;
 import frc.robot.commands.sequences.ExitAndShoot;
 import frc.robot.commands.sequences.StartIntakeAndFeeder;
-import frc.robot.commands.shooter.ShootSpeaker;
 import frc.robot.constants.Constants;
-import frc.robot.subsystems.Climber;
-import frc.robot.subsystems.Deployer;
-import frc.robot.subsystems.Feeder;
-import frc.robot.subsystems.IntakeSubsystem;
-import frc.robot.subsystems.Ramp;
-import frc.robot.subsystems.Shooter;
-import frc.robot.subsystems.SwerveDrivetrain;
+import frc.robot.subsystems.*;
```

Comment:
Is this intellij that keeps wanting to collapse the imports? We'll probably keep flip-flopping this as I believe VSCode prefers explicit listings.

Reply:
I think there is a threshold in intellij before it uses the *. I can look into it today because it does make reviewing PR more annoying.